operator/api/loki/v1/lokistack_types.go

```diff
@@ -310,11 +310,22 @@ type OpenshiftTenantSpec struct {
 type OpenshiftOTLPConfig struct {
 	// DisableRecommendedAttributes can be used to reduce the number of attributes used as stream labels.
 	//
-	// Enabling this setting removes the "recommended attributes" from the generated Loki configuration. This will cause
-	// some stream labels to disappear from the index, potentially making queries more expensive and less performant.
-	//
-	// Note that there is a set of "required attributes", needed for OpenShift Logging to work properly. Those will be
-	// added to the configuration, even if this field is set to true.
+	// Enabling this setting removes the "recommended attributes" from the stream labels. This will make
+	// queries that we using these attributes more expensive and less performant.
```

Comment:
I think the biggest difference is that it changes the actual query. The "expensive" part then comes as a result of this, because more data needs to be searched through because it's no longer separate streams (but as we've already discussed that impact is might be negligible because most labels are just an additional attribute identifying the same stream).